automation_tools/__init__.py

```diff
@@ -1465,3 +1469,178 @@ def setenforce(mode):
         ))
 
     run('setenforce {0}'.format(mode))
+
+
+def get_hostname_from_ip(ip):
+    """Retrives the hostname by logging into remote machine by IP.
+    Specially for the systems who doesnt support reverse DNS.
+    e.g usersys machines.
+
+    :param str ip: Remote host's IP address to retrive hostname.
+    """
+    gethost = lambda: run('hostname')
+    result = execute(gethost, host=ip)
+    print('host', result[ip])
```

Comment:
Actually I used print for debugging and forgot to delete it. Removing print statement now as its not required.